src/Views/Form/defaultFormDonationProcessing.php

```diff
@@ -1,23 +1,13 @@
-<!DOCTYPE html>
-<html <?php language_attributes(); ?>  style="margin-top: 0 !important;">
-	<head>
-		<meta charset="utf-8">
-		<title><?php _e( 'Donation Processing', 'give' ); ?></title>
-		<?php
-		/**
-		 * Fire the action hook in header
-		 */
-		do_action( 'give_embed_head' );
-		?>
-	</head>
-	<body class="give-form-templates">
-		<?php
-		echo apply_filters( 'give_payment_confirm_' . give_clean( $_GET['payment-confirmation'] ), '' );
+<?php
+/**
+ * Payment confirmation view.
+ *
+ * @since 2.7.0
+ */
+use Give\TemplateSkinManager;
 
-		/**
-		 * Fire the action hook in footer
-		 */
-		do_action( 'give_embed_footer' );
-		?>
-	</body>
-</html>
+$tm = new TemplateSkinManager();
```

Comment:
Please avoid short, non-descriptive variable names. Short is nice to type and hard to read and comprehend.

Reply:
resolved